Automattic/jetpack
# Publicize: do not display Twitter in post-publish notice

## Proposed changes:

In the Classic Editor, we display a notice when you publish a post, indicating that your post has been sent to all connected services.

Since Twitter is no longer supported, we should not display it in that message.
<img width="1213" alt="Screenshot 2023-09-14 at 11 09 11" src="https://github.com/Automattic/jetpack/assets/426388/898fa6ff-7234-4263-b47b-6103f645f9bc">

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

**You can only test this on an existing site, where you had connected Publicize to a Twitter account in the past.**

1. Go to Plugins > Add New, install and activate the Classic Editor plugin.
2. Go to Posts > Add New, write a new post.
3. Publish your post.
4. You should not see Twitter mentioned in the post-publish notice.
